### ISSUE TITLE
parsing error if css class contains double dash (--)

### ISSUE THREAD

**Issue by mikethm** [clean-css/clean-css#1223]
Minifying the following gives a blank style.
It appears that if a comment is the first line, the next line is missed if the name has a double dash (e.g. in BEM naming convention)
```
<style>

.double--dash {
    /* the line after this is ignored */
    content: "hi";   
}

</style>
```
![172410724-d1942ada-b631-41ab-bd24-38f03d16d1f0](https://user-images.githubusercontent.com/2066581/172411907-9b87e77a-d9ad-4569-95c8-5e86e97d0fbb.png)
![172410657-4f0107a9-e243-4689-a51f-31f17c5d503a](https://user-images.githubusercontent.com/2066581/172411917-0d7f6db7-be3e-4ed8-be9c-4565ba7da11e.png)

**Comment by boris-petrov** [comment]
@mikethm this is a duplicate of #1218 and has been fixed in `master`.

**Comment by jakubpawlowicz** [comment]
I should push it in 5.3.1.

**Comment by jakubpawlowicz** [comment]
Released in 5.3.1.
